src/azure-cli/azure/cli/command_modules/consumption/custom.py

```diff
@@ -92,13 +92,18 @@ def _output(self, *args, **kwargs):
 
 def transform_usage_output(result):
     from dateutil import parser
-    usageStart = parser.parse(result['usageStart'])
-    usageEnd = parser.parse(result['usageEnd'])
-    result['usageStart'] = usageStart.strftime("%Y-%m-%dT%H:%M:%SZ")
-    result['usageEnd'] = usageEnd.strftime("%Y-%m-%dT%H:%M:%SZ")
-    result['usageQuantity'] = str(result['usageQuantity'])
+    usageStart = result.get('usageStart', None)
+    usageEnd = result.get('usageEnd', None)
+    if usageStart and usageEnd:    
```

Comment:
Is it possible only usageStart or usageEnd returned in the result
```suggestion
    if usageStart or usageEnd:    
```